src/ui/activity-iframe-view.js

```diff
@@ -36,13 +36,15 @@ export class ActivityIframeView extends View {
    * @param {string} src
    * @param {!Object<string, ?>=} args
    * @param {boolean=} shouldFadeBody
+   * @param {boolean=} hasLoadingIndicator
```

Comment:
Let's also include the file where you set it to true.

Reply:
done

Reply:
done